lm_eval/api/registry.py

```diff
@@ -80,6 +80,7 @@ def decorate(fn):
     ],
     "loglikelihood_rolling": ["word_perplexity", "byte_perplexity", "bits_per_byte"],
     "multiple_choice": ["acc", "acc_norm"],
+    "winograd_schema": ["acc", "acc_norm"],
```

Comment:
I think we should default to only “acc” for winograd_schema tasks.

Reply:
Agree. Will make appropriate changes